tests/lib/rules/array-element-newline.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @author alshyra
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/array-element-newline')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('array-element-newline', rule, {
+  valid: [
+    '<template><div :attr="[]" /></template>',
+    '<template><div :attr="[a]" /></template>',
+    '<template><div :attr="[a,\nb,\nc]" /></template>',
```

Comment:
Could you please use template strings (`` `foo` ``) instead of single quotes, and replace `\n` with real newlines? That would be more readable. Indentation shouldn't matter for this rule.

Reply:
Is it better now ?